.travis.yml

```diff
@@ -22,11 +23,12 @@ jobs:
       - cd ../..
       - docker-compose -f docker-compose-dev.yml up -d --build
       - docker run --env-file .env.test --entrypoint poetry todo-app-test:latest run pytest tests
-      - docker run -e MONGO_CONNECTION_STRING -e MONGODB_COLLECTION_NAME -e SECRET_KEY -e GITHUB_CLIENT_ID -e GITHUB_CLIENT_SECRET --entrypoint poetry todo-app-test:latest run pytest tests_e2e
+      - docker run -e MONGO_CONNECTION_STRING -e MONGODB_COLLECTION_NAME -e SECRET_KEY -e GITHUB_CLIENT_ID -e GITHUB_CLIENT_SECRET -e LOG_LEVEL="DEBUG" -e LOGGLY_TOKEN --entrypoint poetry todo-app-test:latest run pytest tests_e2e
```

Comment:
Do you actually want your end-to-end tests logging to your monitoring? You might, but if we were to stick with that longer term I'd definitely want to see some separation per environment!